Formula/conductr-cli.rb

```diff
@@ -0,0 +1,14 @@
+class ConductrCli < Formula
+  desc "The Lightbend ConductR CLI"
+  homepage "https://conductr.lightbend.com"
+  url "https://bintray.com/lightbend/generic/download_file?file_path=conductr-cli-1.0.1-Mac_OS_X-x86_64.zip"
```

Comment:
Ah actually I see that I can drop binaries on to a releases page also. I didn't appreciate that. However I'd prefer to leave the binary at bintray if that's ok given that we have other things referring to it there. Would that be ok with you?